fragdenstaat_de/fds_blog/templates/fds_blog/includes/social_meta.html

```diff
@@ -3,26 +3,14 @@
 {# Schema.org markup for Google+ #}
 <meta itemprop="name" content="{{ article.meta_title }}">
 {% if article.abstract %}<meta itemprop="description" content="{{ article.abstract|striptags }}">{% endif %}
-{# Twitter Card data #}
```

Comment:
why remove all twitter markup? some of these cannot be set from `og:*` meta tags?

Reply:
Which ones?

Reply:
Are services aside from Twitter using them? Otherwise I personally don't mind.